packages/history/src/index.ts

```diff
@@ -5,21 +5,36 @@
 export interface NavigateOptions {
   ignoreBlocker?: boolean
 }
+
+type SubscriberHistoryAction =
+  | {
+      action: HistoryAction | 'ROLLBACK'
```

Comment:
Can this property be renamed to `type`, having to do `action.action` is odd

Reply:
done